docs/docs/guides/manage/self-hosted/configure/docker-compose.yaml

```diff
@@ -1,53 +1,55 @@
-version: '3.8'
+version: "3.8"
 
 services:
-
   zitadel:
-    restart: 'always'
+    restart: "always"
     networks:
-      - 'zitadel'
-    image: 'ghcr.io/zitadel/zitadel:stable'
+      - "zitadel"
+    image: "ghcr.io/zitadel/zitadel:stable"
     command: 'start-from-init --config /example-zitadel-config.yaml --config /example-zitadel-secrets.yaml --steps /example-zitadel-init-steps.yaml --masterkey "${ZITADEL_MASTERKEY}" --tlsMode disabled'
     depends_on:
-      chown:
-        condition: 'service_completed_successfully'
+      certs:
+        condition: "service_completed_successfully"
     ports:
-      - '8080:8080'
+      - "8080:8080"
     volumes:
-      - './example-zitadel-config.yaml:/example-zitadel-config.yaml:ro'
-      - './example-zitadel-secrets.yaml:/example-zitadel-secrets.yaml:ro'
-      - './example-zitadel-init-steps.yaml:/example-zitadel-init-steps.yaml:ro'
-      - 'zitadel-certs:/crdb-certs:ro'
+      - "./example-zitadel-config.yaml:/example-zitadel-config.yaml:ro"
+      - "./example-zitadel-secrets.yaml:/example-zitadel-secrets.yaml:ro"
+      - "./example-zitadel-init-steps.yaml:/example-zitadel-init-steps.yaml:ro"
+      - "zitadel-certs:/crdb-certs:ro"
 
-  chown:
-    image: 'cockroachdb/cockroach:v22.1.0'
-    entrypoint: [ '/bin/bash', '-c' ]
-    command: [ 'cp /certs/ca.crt /zitadel-certs/ && cp /certs/client.root.crt /zitadel-certs/ && cp /certs/client.root.key /zitadel-certs/ && chown 1000:1000 /zitadel-certs/* && chmod 0400 /zitadel-certs/*' ]
+  certs:
+    image: "cockroachdb/cockroach:v22.1.0"
+    entrypoint: ["/bin/bash", "-c"]
+    command:
+      [
+        "cp /certs/* /zitadel-certs/ && cockroach cert create-client --overwrite --certs-dir /zitadel-certs/ --ca-key /zitadel-certs/ca.key zitadel_user && chown 1000:1000 /zitadel-certs/*",
+      ]
     volumes:
-      - 'certs:/certs:ro'
-      - 'zitadel-certs:/zitadel-certs:rw'
+      - "certs:/certs:ro"
+      - "zitadel-certs:/zitadel-certs:rw"
     depends_on:
       my-cockroach-db:
-        condition: 'service_healthy'
+        condition: "service_healthy"
 
   my-cockroach-db:
-    restart: 'always'
+    restart: "always"
     networks:
-      - 'zitadel'
-    image: 'cockroachdb/cockroach:v22.1.0'
-    command: 'start-single-node --advertise-addr my-cockroach-db'
+      - "zitadel"
+    image: "cockroachdb/cockroach:v22.1.0"
+    command: "start-single-node --advertise-addr my-cockroach-db"
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:8080/health?ready=1"]
-      interval: '10s'
-      timeout: '30s'
+      interval: "10s"
+      timeout: "30s"
       retries: 5
-      start_period: '20s'
+      start_period: "20s"
     ports:
-      - '9090:8080'
-      - '26257:26257'
+      - "9090:8080"
+      - "26257:26257"
     volumes:
-      - 'certs:/cockroach/certs:rw'
-      - 'data:/cockroach/cockroach-data:rw'
+      - "certs:/cockroach/certs:rw"
+      - "data:/cockroach/cockroach-data:rw"
```

Comment:
Lets also add the CRDB data dir for persistency here

Reply:
/cockroach/cockroach-data is the data dir... Am I missing something?